Source/MediaStorage.swift

```diff
@@ -447,6 +447,57 @@ public class MediaStorage {
         }
     }
 
+    public func removeMeta(mediaId mediaId: String!, fieldName: String!, completionHandler: MediaStorageError -> Void) {
+        if accessToken == nil {
+            completionHandler(MediaStorageError(statusCode: nil, message: "wrong usage: use the connect method to get an access token."))
+            return
+        }
+
+        var url : String
+        var userMetaKey = ""
+        if fieldName == nil {
+            completionHandler(MediaStorageError(statusCode: nil, message: "invalid parameter: \(fieldName)"))
+            return
+        } else if fieldName == metaUser {
+            // DELETE /media/{id}/meta/user
+            url = "\(mstorageEndpoint)/\(mediaId)\(getUserMetaPath)"
+        } else {
+            userMetaKey = replaceUserMeta(fieldName)
+            if userMetaKey == "" {
```

Comment:
I think  `userMetaKey.isEmpty` is better than `userMetaKey == ""`.
Show here: https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/CollectionTypes.html